tests/config-tests.js

```diff
@@ -4,39 +4,52 @@ const ajv = new Ajv()
 const fs = require('fs')
 const platforms = require('./../platforms')
 
+function formatErrors (errors) {
+    if (!Array.isArray(errors)) {
+        return ''
+    }
+
+    return errors.map(item => `${item.instancePath}: ${item.message}`).join(', ')
```

Comment:
`ajv` returns some error messages that make it easier to track down bugs
![more-info](https://user-images.githubusercontent.com/985504/141807166-2e71cc8a-d7e3-4c9b-b36e-73bf9b523f8d.jpg)